core/lib/src/form/validate.rs

```diff
@@ -596,7 +607,7 @@ pub fn range<'v, V, R>(value: &V, range: R) -> Result<'v, ()>
     };
 
 
-    Err((start, end))?
+    Err((start, end).into())
```

Comment:
At the end of the function, I'm totally okay with an `Err(..)?`. Please revert this and other similar invocations.